src/HistoryService.Validation/Service/CreateServiceRequestValidator.cs

```diff
@@ -0,0 +1,21 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.HistoryService.Data.Interfaces;
+using LT.DigitalOffice.HistoryService.Models.Dto;
+using LT.DigitalOffice.HistoryService.Validation.Service.Interfaces;
+
+namespace LT.DigitalOffice.HistoryService.Validation.Service
+{
+  public class CreateServiceRequestValidator : AbstractValidator<CreateServiceRequest>, ICreateServiceRequestValidator
+  {
+    public CreateServiceRequestValidator(IServiceRepository repository)
+    {
+      RuleFor(service => service.Name.Trim())
```

Comment:
If you get null => nullRefExc. Add cascade.Stop

Reply:
ok